electron/preload.ts

```diff
@@ -56,22 +56,22 @@ async function getProcessesUnix({ wsl = false }): Promise<Process[]> {
 
 	const rows = scriptResult.split(UNIX_NEWLINE).slice(1);
 
-	return rows
-		.map(row => {
-			const columnSeparator = "  ";
-			row = row.trim().replace(/[ ]+/g, columnSeparator);
-
-			const [command, pid, user, fd, type, deviceSize, offset, node, name] = row.split(columnSeparator);
-			const [ipAddress, portNumber] = name.split(":");
-
-			return {
-				command,
-				id: Number.parseInt(pid, 10),
-				portNumber: Number.parseInt(portNumber, 10) || INVALID_PORT_NUMBER_CODE,
-				isWSL: wsl,
-			};
-		})
-		.filter(process => process.portNumber !== INVALID_PORT_NUMBER_CODE);
+	return rows.map((row) => {
+		const columnSeparator = "  ";
+		row = row.trim().replace(/[ ]+/g, columnSeparator);
+
+		const [command, pid, user, fd, type, deviceSize, offset, node, name] = row.split(columnSeparator);
+		const [ipAddress, portNumber] = name.split(":");
+
+		return {
+			command,
+			id: Number.parseInt(pid, 10),
+			// portNumber: Number.parseInt(portNumber, 10) || INVALID_PORT_NUMBER_CODE,
+			portNumber,
```

Comment:
Using a string instead of parsing into a number will probably break functionality for specifying a port range.